spec/support/stubbed_api.rb

```diff
@@ -59,12 +59,12 @@
   end
 end
 
-shared_context "api latest privacy policy" do
-  let(:latest_policy) { build(:api_privacy_policy) }
+shared_context "api current privacy policy" do
+  let(:current_policy) { build(:api_privacy_policy) }
 
   before do
     allow_any_instance_of(GetIntoTeachingApiClient::PrivacyPoliciesApi).to \
```

Comment:
Could have a `with(current_policy.id)` here, but no biggie

Reply:
That will probably go away when we start using the API to serve the privacy policy, so I wanted keep the refactoring to minimum for now :)